src/utils.ts

```diff
@@ -27,13 +27,25 @@ export const extendArray = <T>(arr1: T[], arr2: T[]): T[] => {
   return arr1;
 };
 
+const earliest = (a: number, b: number) => {
+  if (a === -1 && b === -1) return 0;
+  if (a === -1) return b;
+  if (b === -1) return a;
+  return Math.min(a, b);
+};
+
 export const wrapComment = (comment: string, additionalTags: DocumentationTag[] = []): string[] => {
   if (!comment && !additionalTags.length) return [];
   comment = comment.replace(/^\(optional\)(?: - )?/gi, '');
   if (!comment && !additionalTags.length) return [];
   const result = ['/**'];
   while (comment.length > 0) {
-    let index = 0;
+    // Default the cut point to be the first "space" or "newline" character
+    let index = earliest(comment.indexOf(' '), comment.indexOf('\n'));
```

Comment:
question: could we ever get issues if we get Windows-style `\r\n` carriage returns instead of just `\n`?

Reply:
Our linter should ensure we don't have `\r\n` in the docs markdown files